server/plugins/Telegram.js

```diff
@@ -15,12 +15,14 @@ function run(trigger, scope, data, config, callback) {
         } else {
             //Notification formatted in Markdown for pretty notifications
             var notificationText = `*${data.agency} - ${data.alias}*\n` + 
-                                    `Message: ${data.message}`;
+                                    `${data.message}\n` + 
+                                    `[Google Maps](https://maps.google.com/maps?q=${data.message})`;
             
             t.sendMessage({
                 chat_id: tConf.chat,
                 text: notificationText,
-                parse_mode: "Markdown"
+                parse_mode: "Markdown",
+                disable_web_page_preview: "true"
```

Comment:
Again - this doesn't seem to have anything to do with the original PR

Reply:
I agree